aws/resource_aws_appautoscaling_policy_test.go

```diff
@@ -165,6 +189,12 @@ func TestAccAWSAppautoScalingPolicy_dynamoDb(t *testing.T) {
 					resource.TestCheckResourceAttr("aws_appautoscaling_policy.dynamo_test", "scalable_dimension", "dynamodb:table:WriteCapacityUnits"),
 				),
 			},
+			{
+				ResourceName:      "aws_appautoscaling_policy.dynamo_test",
+				ImportState:       true,
+				ImportStateIdFunc: testAccAWSAppautoscalingPolicyImportStateIdFunc("aws_appautoscaling_policy.dynamo_test"),
+				ImportStateVerify: true,
+			},
 		},
 	})
 }
```

Comment:
I coundn't add test step to import resource in `TestAccAWSAppautoScalingPolicy_multiplePoliciesSameName` in `resource_aws_appautoscaling_policy_test.go`. Because `app_awsautoscaling_policy` uses `name` as ID,  `ImportStateVerify` mistakingly choose resources.